adobe_vipm/adobe/client.py

```diff
@@ -355,22 +355,22 @@ def create_preview_order(
             "lineItems": [],
         }
 
-        for item in items:
-            product: AdobeProduct = self._config.get_adobe_product(item["productItemId"])
-            quantity = item["quantity"]
-            old_quantity = item["oldQuantity"]
+        for line in lines:
+            product: AdobeProduct = self._config.get_adobe_product(line["item"]["id"])
```

Comment:
According to [API docs] (https://softwareone.atlassian.net/wiki/spaces/GPTEAM/pages/5222072917/Order+Object) line["item"]["id"] is the MPT id of the item (i.e. the product item ID) not the MPN that is what we need here to lookup the product from the mapping. IMO using platform id instead of vendor ids make the extension hard to maintain. It should be **item.externalIDs.vendor**  right ?

Reply:
Wow, it means I have to get it from the product definition :-(